site/google.jet

```diff
@@ -47,7 +47,7 @@
 {{block googleTagManagerNoScript(tagManagerId=config("google_tag_manager_id"))}}
   {{ if len(tagManagerId) > 0 }}
   <!-- Google Tag Manager (noscript) -->
-  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ tagManagerId }}" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+  <noscript><iframe title="Google Tag Manager" src="https://www.googletagmanager.com/ns.html?id={{ tagManagerId }}" height="0" width="0" style="display:none;visibility:hidden" aria-hidden="true"></iframe></noscript>
```

Comment:
It seems that "[best practice](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute#best_practices)" is to not use `aria-hidden` if already hidden by styling.

Reply:
Yeah I've seen that... but I've also seen people on Stackoverflow saying that this particular best practice needs to be ignored to get around automated WCAG testing.